utils/eslint-config/package.json

```diff
@@ -39,7 +39,7 @@
     "eslint-plugin-testing-library": "^4.12.4"
   },
   "engines": {
-    "node": "^10.12.0 || >=12.0.0"
+    "node": "^10.12.0 || >=14.18.2"
   },
```

Comment:
We can go ahead and remove this engines completely. We should not be building this package from within the package directory. And we should definitely not still allow node 10.

Reply:
Engine removed from package.